backend/edge-full/src/main/kotlin/io/featurehub/edge/stats/StatsCollectionOrchestrator.kt

```diff
@@ -24,9 +24,9 @@ class StatsCollectionOrchestrator @Inject constructor(private val publisher: Sta
   var maxApiKeysPerPublish: Long? = 300
 
   companion object Prometheus {
-    val publishTimeHistogram = Histogram.build("edge_publish_time", "Time taken to publish stats to NATS").register()
-    val successfulPublishing = Counter.build("edge_publish_success", "Number of successes for publishing").register()
-    val failedPublishing = Counter.build("edge_publish_failure", "Number of failures for publishing").register()
+    val publishTimeHistogram: Histogram = Histogram.build("edge_publish_time", "Time taken to publish stats to NATS").register()
```

Comment:
why declare the types here? they are implicit from the return value?

Reply:
The return type is not implicit. See the warning in IntelliJ IDE

<img width="1403" alt="IDE_Warning" src="https://user-images.githubusercontent.com/38799/167233381-4788b44d-35c9-464b-ad7f-4c13a3807fb9.png">

Reply:
Right, that makes sense now.